### PR TITLE
AISDK-105: Clarfiy mentions of balance in Node SDK

### PR DESCRIPTION
**Pull request by KostasRev**

### CODE REVIEW

**Review by k-weng — APPROVED** [review]
:shipit:

File: examples/async_transcribe_local_media_file.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const token = require('./config/config.json').access_token;
     // Get account details
     var account = await client.getAccount();
     console.log(`Account: ${account.email}`);
-    console.log(`Balance: ${account.balance_seconds} seconds`);
+    console.log(`Credits remaining: ${account.balance_seconds} seconds`);
```
Comment (k-weng) [inline examples/async_transcribe_local_media_file.js:12]: Should we change `balance_seconds` to `credits_seconds` or something similar?

Comment (KostasRev) [reply examples/async_transcribe_local_media_file.js:12]: I believe not, we only want to change strings and descriptions, not any code.

Comment (amikofalvy) [reply examples/async_transcribe_local_media_file.js:12]: Yes. No breaking changes. thanks

**Pull request merged**